README.md

```diff
@@ -180,23 +180,23 @@ depending on whether you want to use HTTPS or SSH. Finally, run the
 following:
 
 ```sh
-
 python3 -m pip install -r requirements.txt
 ```
 
 ## Usage
 
 To run your first example Scene, you can run the following commands:
 For users:
-1. Download the `example_scenes.py` file from [GitHub](https://github.com/ManimCommunity/manim/blob/master/example_scenes.py)
+
+1. Download the `example_scenes.py` file from [GitHub](https://github.com/ManimCommunity/manim/blob/master/example_scenes/basic.py)
 2. Run the following command:
 ```sh
-manim example_scenes.py SquareToCircle -pl
+manim basic.py SquareToCircle -pl
```

Comment:
It's `example_scenes/basic.py`

Reply:
When downloaded the user will store like that.

Reply:
huh? @naveen521kk